src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page">
+    <div class="card" data-qa="card">
+
+      <img
+        src="images\imac.jpeg"
+        alt="product-image"
+        class="card__img"
+      >
+
+      <div class="main-text">
+        <p class="main-text__name">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+        <p class="main-text__number">
+          Код товара: 195434
+        </p>
+      </div>
```

Comment:
It's so little block to reuse. That's why you should change it to card's element, and also need to change classes in styles.

Please use more understandable naming of classes: title, code - for example.

```suggestion
      <div class="card__main-text">
        <p class="card__title">
          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
        </p>
        <p class="card__code">
          Код товара: 195434
        </p>
      </div>
```